pkg/state/state_ofed.go

```diff
@@ -829,8 +845,8 @@ func (s *stateOFED) handleCertConfig(
 func (s *stateOFED) handleSubscriptionVolumes(
 	ctx context.Context, osname string, runtime string, mounts *additionalVolumeMounts) error {
 	reqLogger := log.FromContext(ctx)
-	if osname == "rhel" && runtime != nodeinfo.CRIO {
-		reqLogger.V(consts.LogLevelDebug).Info("Setting subscription mounts for RHEL with non CRIO container runtime")
+	if osname == "rhel" && runtime != nodeinfo.CRIO || osname == "sles" {
```

Comment:
.nit For code readability improvement
```suggestion
	if (osname == "rhel" && runtime != nodeinfo.CRIO) || osname == "sles" {
```

Reply:
Done